### DISCUSSION TITLE
MS SQL SERVER

### DISCUSSION THREAD

**Discussion by jusce17** [yezz123/ormdantic#16] [Q&A] [answered]
is it possible to use it with MS SQL SERVER ?

**Comment by yezz123** [comment, marked as answer]
If it's supported with an asynchronous driver that serves the case yes it's possible! Check the drivers supported by SQLAlchemy!

- Engine Support: https://docs.sqlalchemy.org/en/14/core/engines.html
- Support for the Microsoft SQL Server database in SQLAlchemy: https://docs.sqlalchemy.org/en/14/dialects/mssql.html
